### DISCUSSION TITLE
Why doesn't <a> target change?

### DISCUSSION THREAD

**Discussion by ShockedCoder** [bs-community/blessing-skin-server#387] [Q&A] [answered]
I do not want my skin server to open a new tab every time to preview a skin.
I first tried removing all instances of `target="_blank"`, but it still came back somehow.
So then I tried replacing `target="_blank"`with `target="_self" (This is the default value)`, but somehow it still didn't change.

I've tried all I can, I've even manually gone through the files in case my replacer missed something, but to no avail.
I'd really appreciate the help.

**Comment by Big-Cake-jpg** [comment]
Please make sure you changed right TWIG template.

**Reply by ShockedCoder** [reply]
I changed everything. I used CTRL+SHIFT+F to find every instance of it using vscode

**Comment by mochaaP** [comment, marked as answer]
Clean template cache and retry.
p.s. it's advised to use override than in-place replace.

**Reply by ShockedCoder** [reply]
How do I "override"? And where is the cache located?

**Reply by mochaaP** [reply]
Copy `/resources/views/**` to `/resources/views/overrides/**` and do edits.

Cleaning view cache: `php artisan view:clear`
